### PR TITLE
Improve package health: better description; add an example

### PR DESCRIPTION
**Pull request by srawlins**
Should help to boost our score.

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: expected_output
 version: 1.2.2-dev
-description: Easily write and test multiline input and expected output
+description: >
```
Comment (kevmoo) [inline pubspec.yaml]: Use `>-` to collapse inner newlines, too!

**Review by kevmoo — APPROVED** [review]

File: example/example.dart [hunk 2]
```diff
@@ -0,0 +1,13 @@
+library touppercase.tests;
```
Comment (kevmoo) [inline example/example.dart:5]: Add copyright header? Remove library?

Comment (srawlins) [reply example/example.dart:5]: Library is needed for reflection. Added copyright header.

File: pubspec.yaml (hunk 1 above)
Comment (srawlins) [reply pubspec.yaml]: Done!

**Pull request merged**